sdcm/nemesis.py

```diff
@@ -2811,7 +2818,8 @@ def execute_data_validation_thread(command_template, keyspace_name, number_of_ro
             self.log.info("Restoring the schema of the keyspace '%s'", chosen_snapshot_info["keyspace_name"])
             restore_task = mgr_cluster.create_restore_task(restore_schema=True, location_list=location_list,
                                                            snapshot_tag=chosen_snapshot_tag)
-            restore_task.wait_and_get_final_status(step=10, timeout=120)
+            step = 10
+            restore_task.wait_and_get_final_status(step=step, timeout=chosen_snapshot_info['expected_timeout'] / step)
```

Comment:
This hunk looks unrelated

Reply:
well, it is related to the fact that the test is still not passing, and now it is failing on this call, timing out while waiting for this task to end... but now i have moved it to its own commit